odh-dashboard/base/kustomization.yaml

```diff
@@ -28,4 +28,4 @@ images:
   newTag: v2.12.0
 - name: oauth-proxy
   newName: registry.redhat.io/openshift4/ose-oauth-proxy
```

Comment:
let me know what you think about adding a yaml comment line for humans to read on the tag version used

i.e.

# tag v4.10 as of July 6 2023